docarray/typing/tensor/torch_tensor.py

```diff
@@ -230,3 +230,7 @@ def get_comp_backend() -> Type['TorchCompBackend']:
         from docarray.computation.torch_backend import TorchCompBackend
 
         return TorchCompBackend
+
+    @classmethod
+    def __torch_function__(cls, func, types, args=(), kwargs=None):
+        return super().__torch_function__(func, (torch.Tensor,), args, kwargs)
```

Comment:
what I don't get is because it does not recognize this from the mother class. Maybe because of the weird metaclass stuff they have. Though this looks like a bug in pytorch